CarRentals/CarRentals/Controllers/CarCRUD.cs

```diff
@@ -13,7 +13,7 @@ namespace CarRentals
 {
     public class CarCRUD : IDataProcessing
     {
-        string PATH = ConfigurationManager.AppSettings["JsonFile"].ToString();
+        string Path = Program.JsonFilePath();
```

Comment:
ultra minor: unless this is a public property, by code convention it should start with lowercase and with a `_` as a prefix (check https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#camel-case)

Reply:
Thanks! I have corrected it. However I'm not happy with this, because I think that you want I use dependency injection. I tried to watch a lot of videos and read a lot of post and the Microsoft documentation, but I couldn't achieve this in a console app. I'm not going to give up, I will continue trying. Anyway, when I will change the console for a web application, it will be easyest

Reply:
@jrocaspana I have created a new pull request with a little change in the project. And now I think that is much better. 
This is the link: 
https://github.com/manu3210/MSAcademy/pull/2

Please, could you tell me what you think?.

Reply:
sure! I will check today later in the morning and give you feedback, we can also discuss this topic today in our meeting